runtime/oti/util_api.h

```diff
@@ -1375,6 +1375,18 @@ J9UTF8*
 preloadClassNameAtIndex(U_32* permittedSubclassesCountPtr, U_32 index);
 #endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
 
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+/**
+ * Retrieves flags stored in ImplicitCreation attribute. This method assumes
+ * there is an ImplicitCreation attribute in the ROM class.
```

Comment:
Might be better to mention it should be called only when `J9_ROMCLASS_OPTINFO_IMPLICITCREATION_ATTRIBUTE` is set in `romClass->optionalFlags`